src/how-to/create-dates.md

```diff
@@ -0,0 +1,87 @@
+# Create dates and times
+
+`time` provides a [`const`](https://doc.rust-lang.org/std/keyword.const.html)-ready API. All 
+functions here are `const`: values are calculated at compile time, not at runtime.
+
+For convenience, macros are provided with
+[feature `macros`](https://docs.rs/time/latest/time/#feature-flags). They let us restrict
+parameters further, avoiding the need to `unwrap()` at the cost of compilation time.
+
+## Creating [`Date`](https://docs.rs/time/latest/time/struct.Date.html)s
+
+From a constant value, with [feature `macros`](https://docs.rs/time/latest/time/#feature-flags):
+
+```rust
+use time::macros::date;
+
+let _ = date!(2022-01-02);
+```
+
+From a calendar date:
+```rust
+use time::{Date, Month};
+
+let _ = Date::from_calendar_date(2022, Month::January, 2).unwrap();
+```
+
+## Creating [`PrimitiveDateTime`](https://docs.rs/time/latest/time/struct.PrimitiveDateTime.html)s
+
+A `PrimitiveDateTime` is both a date and a time. We can create them directly:
+
+```rust
+use time::macros::datetime;
+
+let _ = datetime!(2022-01-02 11:12:13.1_234_567_890);
```

Comment:
```suggestion
let _ = datetime!(2022-01-02 11:12:13.123_456_789);
```

Digits after the ninth are discarded, as `time` supports nanosecond precision. Other lines that use similar values should be changed as well.